Lib/idlelib/help_about.py

```diff
@@ -2,14 +2,23 @@
 
 """
 import os
-from platform import python_version
+import sys
+from platform import python_version, architecture
 
 from tkinter import Toplevel, Frame, Label, Button, PhotoImage
 from tkinter import SUNKEN, TOP, BOTTOM, LEFT, X, BOTH, W, EW, NSEW, E
 
 from idlelib import textview
 
 
+def _bitness():
+    "Return bitness of platform."
+    bits, linkage = architecture()
+    if sys.platform == 'darwin':
+        bits = '64bit' if sys.maxsize > 2**32 else '32bit'
```

Comment:
if sys.platform == 'darwin':
        return '64' if sys.maxsize > 2**32 else '32'
    else:
        return  architecture()[0][:2]

I guess I prefer multiple crisp returns to multiple assignments used to make a delayed return.  I do understand the single exit idea from the Structured Programming revolution of the 1970s.  'Single exit' replaced multiple *unrestricted* GOTOs.  A return is a very disciplined goto.

Since I have 64 bit installs and 32 bit clone debug builds, I tested and confirmed that the return on windows really is build bits and, in spite of the name, not physical architecture bits.